radix-engine/src/blueprints/epoch_manager/events/epoch_manager.rs

```diff
@@ -8,6 +8,13 @@ pub struct RoundChangeEvent {
 
 #[derive(ScryptoSbor, ScryptoEvent, PartialEq, Eq)]
 pub struct EpochChangeEvent {
+    /// The *new* epoch's number.
     pub epoch: u64,
+    /// The *new* epoch's validator set.
     pub validators: BTreeMap<ComponentAddress, Validator>,
+    /// A sum of staked XRD amounts of the entire validator set of the *concluded* epoch.
+    /// Note: calculating `EpochManagerConfigSubstate.total_emission_xrd_per_epoch / validator_set_stake_xrd`
+    /// gives a good estimate on an expected gains per 1 XRD staked per epoch (ignoring validator
```

Comment:
_(note from author)_

@dhedey You requested this from me explicitly (i.e. "a sense of what the APY is, ignoring validator performance"), but now, after adding, I noticed that actually the same information can currently be easily obtained by summing up the `EpochChangeEvent.validators.map(|v| v.stake).sum()` [of the previous epoch's event]).

Are you ok with dropping this `validator_set_stake_xrd`?

Reply:
> But as you say, you can get that by summing the `ValidatorEmissionEvent`s so maybe it's not so needed.

Actually I claim you can get the `validator_set_stake_xrd` from the single `EpochChangeEvent` - namely from its existing field `validators` (each one has `.stake` there).

(and regarding the `EpochEmissionEvent. total_emitted_xrd` I think that existing `EpochManagerConfigSubstate.total_emission_xrd_per_epoch` is very close, and maybe even better)

So: let me drop my `EpochChangeEvent` changes 👍